website/blog/2025-06-06-the-new-dbt-vscode-extension.md

```diff
@@ -0,0 +1,109 @@
+
+Since the acquisition of SDF Labs by dbt Labs, I have been waiting for the day that we would see what the result of the fusion of these two companies would be. Spoiler alert: it’s the dbt Fusion engine and its better than I could have expected.
+
+## The dbt developer experience in the pre-fusion-era
+
+If you've ever started a dbt project, chances are your journey began like mine did: cloning jaffle_shop, opening it in VS Code, and running dbt run for the first time (actually the second time, because I know you forgot to run dbt deps in the first one). It is like our initiation process, our ‘hello-world’.
+
+You played around with staging models, the orders table, customers table. But let's be honest, the developer experience in that setup was always a bit... clunky.
+
+You wanted to check the lineage of your project, one of the coolest features of dbt, and you had to run dbt docs generate, serve, and open the docs in a browser. Made some updates? Do all the steps again.
+
+Did you want to check your project's metadata? You had to rely on dbt docs (that whole process again), or build some custom solution with the manifest.json.
+
+Moving to dbt Cloud made things smoother. It has a built-in IDE with git integration, easier to compile and preview models. An auto-updating lineage tab below the model, a much better documentation with dbt Explorer, now renamed to Catalog. And a lot of other powerful features for orchestration, observability, CI/CD, and more.
+
+dbt Cloud was a big step up, but even with dbt Cloud, many of us still preferred to use our own dev environments. We like using our themes, our VS Code extensions, our terminals, but this would mean losing all the nice cloud features while developing. A sad trade-off.
+
+We've already been to Cloud and back to the terminal, and some problems remain. No matter where you were writing dbt, if you modified some model, and hadn't noticed you forgot to put a comma (because you are using trailing commas instead of leading commas, and they're harder to see, and I'm talking too much about the comma fight), you waited dbt to communicate with your DW, waited your cluster to start, and a minute later you got an error saying you should start using leading commas.
+
+All this back-and-forth communication of dbt and the platform was slowing down your project.
+
+That’s why this new release is such a big deal. It solves all the problems above and introduces other things I didn't know I needed until I saw it.
+
+## The new era of dbt development
+
+With the acquisition of SDF Labs and a renewed focus on developer experience, dbt Labs announced their new engine, so goodbye dbt Core, and enter the dbt Fusion engine. This engine was built from zero with Rust, and this is now the intelligent engine that will power up dbt no matter where you run it. There are different ways you can use the Fusion engine, and the best one is with the also announced VSCode Extension.
```

Comment:
I think we need to be careful with this language because dbt Core isn't going anywhere
```suggestion
With the acquisition of SDF Labs and a renewed focus on developer experience, dbt Labs announced its new engine, Fusion. This engine was built from zero with Rust, and this is now the intelligent engine that will power up dbt, no matter where you run it. There are different ways you can use the Fusion engine, and the best one is with the also announced VS Code extension.
```